### PR TITLE
Updated execute file

### PR DESCRIPTION
**Pull request by Devashish-Thakar**
Following updates made to the execute file:

1. To make life simpler for now, removed the population dynamic. We can add this later once we are done with solving for S period problem with endogenous labor.

2. Updated the execution of SS solution with correct functional inputs ( guesses for r, w and param list)

Request all to look at the initial guess value used for w specifically

### CODE REVIEW

File: scripts/execute.py [hunk 1]
```diff
@@ -4,32 +4,32 @@
...
+'''
+Provided a guess of w = 0.05. Request to cross check if there is 
+any better guess that can be used
+
+We still need to add the TPI file and then update its execution here
+'''
+
+ss_params = (beta, sigma, alpha, A, delta, xi)
+r_init = 1 / beta - 1
+w_init = 0.5 # Not sure if there is a possible way to guess
```
Comment (jdebacker) [inline scripts/execute.py:28]: Why do you need a `w` guess?  If guess `r`, don't you know the `w` that is consistent with that?

Comment (Devashish-Thakar) [reply scripts/execute.py:28]: I am looking at the SS file and w_init as the input

But let me make sure once again

Comment (Devashish-Thakar) [reply scripts/execute.py:28]: It does seem like we need an initial guess of r and w to solve the HH problem
--> which gives b and n

Comment (jdebacker) [reply scripts/execute.py:28]: @devashish2019 But see the SS solution - `w` is found within the while loop.  Only `r_init` is passed to the `SS.solve_ss()` function.

Comment (Devashish-Thakar) [reply scripts/execute.py:28]: @jdebacker I am looking at the SS file which is on the master and it has w_init as an argument.

Comment (Devashish-Thakar) [reply scripts/execute.py:28]: The while look starts with the initial w guess (w = w_init) which is outside the for loop

**Comment by jdebacker** [comment]
@devashish2019 Let me know if this PR should be merged (after updating with changes to the master) or if you'd rather just close it given other changes to the master.

**Comment by Devashish-Thakar** [comment]
I will close it for now and try to get the new changes from the master

**Pull request closed without merging**